icu4c/source/data/brkitr/root.txt

```diff
@@ -22,8 +22,8 @@ root{
         Mymr:process(dependency){"burmesedict.dict"}
         Thai:process(dependency){"thaidict.dict"}
     }
-    lstm{
+   lstm{
         Thai{"Thai_graphclust_model4_heavy.res"}
         Mymr{"Burmese_graphclust_model5_heavy.res"}
-    }
+    }    
```

Comment:
Funky whitespace reformatting. Is this from the CLDR-to-ICU converter?

Reply:
It could just be my editor!

Reply:
Is it a manual edit? If it's manual, I would expect it to be aligned nicely...

Reply:
It is copy and paste of text from section 5e of the integration instructions: https://unicode-org.github.io/icu/processes/cldr-icu.html#5-generate-cldr-production-data-and-convert-for-icu